tools/tertiary-analysis/pyscenic/pyscenic_ctx.xml

```diff
@@ -0,0 +1,142 @@
+<tool id="pyscenic_ctx" name="PySCENIC CTX" profile="21.09" version="0.12.1+galaxy0">
+    <description>
+        computes active regulons based on a gene regulatory network
+    </description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <command><![CDATA[
+    PySCENIC_DB=db.genes_vs_motifs.rankings.feather &&
+    ln -s '${module_fname}' tf2targets.tsv &&
+    ln -s '${expression_mtx}' expr_mat.loom &&
+    ln -s '${database_fname}' \${PySCENIC_DB} &&
+
+    pyscenic ctx tf2targets.tsv \${PySCENIC_DB} 
```

Comment:
```suggestion
    ln -s '${database_fname}' ${PySCENIC_DB} &&

    pyscenic ctx tf2targets.tsv ${PySCENIC_DB} 
```

Reply:
I would think that we still need the && there....
```suggestion
    ln -s '${database_fname}' ${PySCENIC_DB} &&

    pyscenic ctx tf2targets.tsv ${PySCENIC_DB} 
```

Reply:
Thanks!